core-bundle/contao/templates/twig/content_element/accordion.html.twig

```diff
@@ -0,0 +1,36 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+
+{% block content %}
+    {% for element in elements %}
+        <h3 class="handorgel__header">
+            <button class="handorgel__header__button">{{ element.header }}</button>
+        </h3>
+        <div class="handorgel__content"{% if element.is_open %} data-open{% endif %}>
+            <div class="handorgel__content__inner">
+                {{ content_element(element.reference) }}
+            </div>
+        </div>
```

Comment:
```suggestion
        {% block element %}
            {% block element_header %}
                <h3{{ attrs(accordion_header|default).addClass('handorgel__header') }}>
                    <button{{ attrs(accordion_header_button|default).addClass('handorgel__header__button') }}>
                        {{- element.header -}}
                    </button>
                </h3>
            {% endblock %}
            {% block element_content %}
                <div{{ attrs(accordion_content|default).addClass('handorgel__content') }}>
                    <div{{ attrs(accordion_content_inner|default).addClass('handorgel__content__inner') }}>
                        {{ content_element(element.reference) }}
                    </div>
                </div>
            {% endblock %}
        {% endblock %}
```

Reply:
Where is the `data-open` attribute in your suggestion?

Reply:
Adding `.set('data-open', element.is_open)` to the h3 should do the trick I think:
```twig
<h3{{ attrs(accordion_header|default).addClass('handorgel__header').set('data-open', element.is_open) }}>
```

Reply:
It needs to be applied to the `<div class="handorgel__content"` element according to the docs: https://github.com/oncode/handorgel#markup